File: app/src/main/java/com/cogu/spylook/mappers/GrupoToCardItem.java
package com.cogu.spylook.mappers;

import com.cogu.spylook.model.Grupo;
import com.cogu.spylook.model.cards.GrupoCardItem;

import org.mapstruct.Mapper;

@Mapper
public interface GrupoToCardItem {
    GrupoCardItem toCardItem(Grupo grupo);
}

File: app/src/main/java/com/cogu/spylook/mappers/GrupoToCardItem.kt
package com.cogu.spylook.mappers

import com.cogu.domain.model.Grupo
import com.cogu.spylook.model.cards.GrupoCardItem
import org.mapstruct.Mapper
import org.mapstruct.Mapping

@Mapper
interface GrupoToCardItem {
    @Mapping(target = "clickable", constant = "true")
    fun toCardItem(grupo: Grupo): GrupoCardItem
}
